src/index.html

```diff
@@ -17,6 +17,10 @@
     />
```

Comment:
It seems like you are trying to link a SCSS file directly into HTML. Browsers do not understand SCSS syntax, so you should compile your SCSS to CSS and link the resulting CSS file instead.